string input_str
---
SegmentedPointCloud segmented_scene
